SonarQube.Client.Tests/SonarQubeService_GetSourceCodeAsync.cs

```diff
@@ -0,0 +1,89 @@
+﻿/*
+ * SonarQube Client
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Security;
+using System.Threading;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using SonarQube.Client.Models;
+using SonarQube.Client.Tests.Infra;
+
+namespace SonarQube.Client.Tests
+{
+    [TestClass]
+    public class SonarQubeService_GetSourceCodeAsync : SonarQubeService_TestBase
+    {
+        [TestMethod]
+        public async Task Get_V5_0_ExampleFromSonarQube()
+        {
+            await ConnectToSonarQube("5.0.0.0");
+
+            const string sourceCode = @"package org.sonar.check;
+
+public enum Priority {
+/**
```

Comment:
I was extremely confused by this comment and didn't understand why is there an enum declaration in the middle of the test.
Can we go with a simpler source code, one that doesn't have comments in it?